service/rpc/errors.go

```diff
@@ -61,10 +61,12 @@ func LogErrUnimplemented(rosettaEndpoint string) *types.Error {
 }
 
 func LogErrDetails(rosettaErr *types.Error, err error) *types.Error {
-	rosettaErr.Details = map[string]interface{}{
+	copyErr := &types.Error{}
+	*copyErr = *rosettaErr
```

Comment:
this syntax scared me but I googled and apparently this is standard go practice 🤷‍♂️

Reply:
I feel that 😂 was surprised myself when googling lol